root/tree/dataframe/RDataFrameOpenDataBenchmarks.cxx

```diff
@@ -0,0 +1,502 @@
+#include "ROOT/RDataFrame.hxx"
+#include "ROOT/RVec.hxx"
+#include "RConfigure.h" // for R__USE_IMT
+#include "benchmark/benchmark.h"
+#include "rootbench/RBConfig.h"
+#include "Math/Vector4D.h"
+
+
+/*
+ * The original benchmark code and also the original input file can be found here:
+ * https://github.com/stwunsch/opendata-benchmarks
+ */
+
+static const std::string filepath1M = RB::GetDataDir() + "/Run2012B_SingleMu_1M.root";
+static const std::string filepath10M = RB::GetDataDir() + "/Run2012B_SingleMu_10M.root";
+
+void set_nthreads(unsigned int nthreads) {
+#ifdef R__USE_IMT
+   if (nthreads) {
+      ROOT::EnableImplicitMT(nthreads);
+   } else {
+      ROOT::DisableImplicitMT();
+   }
+#endif // R__USE_IMT
+}
+
+// Benchmark 1
+
+void benchmark1_jitted(unsigned int nthreads) {
+    set_nthreads(nthreads);
+    ROOT::RDataFrame df("Events", filepath10M);
+    auto h = df.Histo1D({"", ";MET (GeV);N_{Events}", 100, 0, 2000}, "MET_sumet");
+    *h;
+}
+
+static void BM_RDataFrame_OpenDataBenchmark1_jitted(benchmark::State &state)
+{
+   for (auto _ : state) {
+      const auto nthreads = state.range(0);
+      benchmark1_jitted(nthreads);
+   }
+}
+BENCHMARK(BM_RDataFrame_OpenDataBenchmark1_jitted)->Unit(benchmark::kMillisecond)->Repetitions(1)->Arg(0);
+BENCHMARK(BM_RDataFrame_OpenDataBenchmark1_jitted)->Unit(benchmark::kMillisecond)->Repetitions(1)->Arg(8);
```

Comment:
here and below I think this can be just

```suggestion
BENCHMARK(BM_RDataFrame_OpenDataBenchmark1_jitted)->Unit(benchmark::kMillisecond)->Repetitions(1)->Arg(0)->Arg(8);
```
?

Reply:
I didn't know you can chain different args! I'll give it a try :)